hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -182,6 +182,7 @@
     public static final String PROP_LOGGING_TYPE = "hazelcast.logging.type";
     public static final String PROP_ENABLE_JMX = "hazelcast.jmx";
     public static final String PROP_ENABLE_JMX_DETAILED = "hazelcast.jmx.detailed";
+    public static final String PROP_JMX_UPDATE_INTERVAL_SECONDS = "hazelcast.update.jmx.interval.seconds";
```

Comment:
BTW, this should not be added here anymore. All those String constants are deprecated. Just add an entry to `GroupProperty`.